NodeJS/__integration__/AddressBookClient.it.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  AddressBookClient,
+  createAddressBookClient
+} from '../src/AddressBookClient'
+import { log4js } from '../src/Constants'
+
+const rootLogger = log4js.getLogger()
+rootLogger.level = 'debug'
+rootLogger.debug('Starting AddressBookClient Integration Test')
+
+describe('SubscriptionClient.ts', () => {
+  const localEnv = {
+    user: 'sdetSubAccAdmin',
+    token:
+      'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJzZGV0U3ViQWNjQWRtaW4iLCJpc3MiOiJBQkNfU0VDVVJJVFlfR0FURVdBWSJ9.0HacDx5e1mS9uFmZzrQ8MG8qYp_w27VbsY_z07Tw45w',
+    url: 'https://integration.bpo.avaya.com',
+    subAccountAppId: 'SDE_SDBRec'
+  }
+
+  const { subAccountAppId, token, url } = localEnv
+
+  let addressBookClient: AddressBookClient
+  beforeEach(async () => {
+    addressBookClient = await createAddressBookClient(url, token)
+  })
+
+  test('getAddressBook should return AddressBookPayload with active contacts', async () => {
```

Comment:
Your test says (or at least implies) that all contacts should be "active", but the actual test does not assert that. You will need to update your test or it's name.

Reply:
renamed